chart/templates/deployment.yaml

```diff
@@ -108,14 +106,14 @@ spec:
             - containerPort: 8080
           resources:
             requests:
-              memory: "32Mi"
-              cpu: "50m"
+              memory: "16Mi"
+              cpu: "25m"
             limits:
-              memory: "128Mi"
-              cpu: "500m"
+              memory: "32Mi"
+              cpu: "100m"
           env:
             - name: PUBSUB_TOPIC
-              value: 'sh.keptn.event.test.triggered'
+              value: 'sh.keptn.>'
```

Comment:
Should the PUBSUB_TOPIC also be configureable in `.Values`, since `sh.keptn.>` won't work when the job-executor runs in an execution plane?

Reply:
Thanks for bringing this up, I wasn't aware of this limitation of the execution plane.

Reply:
I have clarified with @johannes-b @agrimmer and @thisthat that this is a known limitation of Keptn execution-plane.

Therefore I will, as you suggested, create a config entry for it, so it can be overwritten using helm values.

Reply:
done